__tests__/app.test.js

```diff
@@ -44,5 +43,31 @@ describe("/api", () => {
                 })
             })
         })
+        describe("/articles/:id", () => {
+            test("returns object with correct keys", () => {
+                return request(app)
+                .get("/api/articles/1")
+                .expect(200)
+                .then(({body}) => {
+                    const { article } = body
+                    const ArtKeys = Object.keys(article[0])
+                    expect(Object.keys(article).length).toBeGreaterThan(0)
+                    expect(ArtKeys).toContain("author")
+                    expect(ArtKeys).toContain("title")
+                    expect(ArtKeys).toContain("article_id")
+                    expect(ArtKeys).toContain("body")
+                    expect(ArtKeys).toContain("topic")
+                    expect(ArtKeys).toContain("created_at")
+                    expect(ArtKeys).toContain("votes")
+                    expect(ArtKeys).toContain("article_img_url")
+                    
```

Comment:
I would be more specific with the test - if you are receiving one article then we should know what it looks like. At the minimum I would check that the article_id is 1 - proving you have the correct article back. I would also check the types of values and not just the keys - you could do a .toEqual on the whole article